config.lua.dist

```diff
@@ -31,6 +33,8 @@ removeChargesFromRunes = true
 removeChargesFromPotions = true
 removeWeaponAmmunition = true
 removeWeaponCharges = true
+removeBeginningWeaponAmmunition = true
```

Comment:
Start config with false. If I want to enable, I'll put true.
```suggestion
removeBeginningWeaponAmmunition = false
```

Reply:
Default should be true since we want to remove ammunition by default, exactly like removeWeaponCharges.